package.json

```diff
@@ -52,10 +52,10 @@
   ],
   "license": "Apache-2.0",
   "peerDependencies": {
-    "@hapi/hapi": ">=18.3.1"
+    "@hapi/hapi": ">=19.2.0"
```

Comment:
We should loosen this to `">=19.0.0", unless there is a specific reason to require `19.2.0`

Reply:
Should probably go back to catalyst-render too to ensure it's consistent there too.